### PR TITLE
Add tests for rendering regions

### PR DESCRIPTION
**Pull request by BenjaminSchaaf**
Also a fix for `middle` being renamed to `center`.

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/246752300)

# Firefox (nightly) #
Testing web-platform-tests at revision 018d2f9f346abfda873efa753dfac77eb2922d11
Using browser at version BuildID 20170612100241; SourceStamp 27cad9749cddf68e11fdd4e5d73dad84a8f8cf23
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>13 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center.html">/webvtt/rendering/cues-with-video/processing-model/align_center.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html">/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/basic.html">/webvtt/rendering/cues-with-video/processing-model/regions/basic.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html">/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/246752301)

# Sauce (safari) #
Testing web-platform-tests at revision 018d2f9f346abfda873efa753dfac77eb2922d11
Using browser at version 10.0
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>13 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center.html">/webvtt/rendering/cues-with-video/processing-model/align_center.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html">/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/basic.html">/webvtt/rendering/cues-with-video/processing-model/regions/basic.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html">/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/246752307)

# Chrome (unstable) #
Testing web-platform-tests at revision 018d2f9f346abfda873efa753dfac77eb2922d11
Using browser at version 61.0.3135.4 dev
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>13 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center.html">/webvtt/rendering/cues-with-video/processing-model/align_center.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_gt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html">/webvtt/rendering/cues-with-video/processing-model/align_center_position_lt_50_size_gt_maximum_size.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html">/webvtt/rendering/cues-with-video/processing-model/align_center_wrapped.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/basic.html">/webvtt/rendering/cues-with-video/processing-model/regions/basic.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/regionanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html">/webvtt/rendering/cues-with-video/processing-model/regions/single_line_top_left.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_x_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/viewportanchor_y_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6336/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html">/webvtt/rendering/cues-with-video/processing-model/regions/width_50_percent.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | TIMEOUT |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/246752308)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 018d2f9f346abfda873efa753dfac77eb2922d11
Using browser at version 14.14393
Starting 10 test iterations

**Review by silviapfeiffer — APPROVED** [review]
LGTM

File: webvtt/rendering/cues-with-video/processing-model/regions/support/scroll_up.vtt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+REGION
+id:1
+lines:2
+regionanchor:0%,0%
+scroll:up
+
+00:00:00.000 --> 00:00:05.000 region:1
+This is a first test subtitle
+
+00:00:01.000 --> 00:00:05.000 region:1
```
Comment (zcorpan) [inline webvtt/rendering/cues-with-video/processing-model/regions/support/scroll_up.vtt]: Doesn't this cue (and the one below) need to start at 00:00:00.000 to show up when `takeScreenshot()` runs? Or should the test use `takeScreenshotDelayed(2000)`? Or am I missing something here? :-)

Comment (silviapfeiffer) [inline webvtt/rendering/cues-with-video/processing-model/regions/support/scroll_up.vtt]: You're right: If the screenshis meant to contain all the cues from the vtt for demonstrating scrolling, it should be delayed until they are all on screen.

**Comment by Ms2ger** [comment]
@BenjaminSchaaf do you have time to address the comment?

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19474)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/287829304)

**Review by silviapfeiffer — APPROVED** [review]
LGTM

**Comment by silviapfeiffer** [comment]
@zcorpan are you still able to review? Or how will we get test changes to WebVTT landed now?

**Review by zcorpan — APPROVED** [review]

**Comment by Ms2ger** [comment]
Looks like all we need now is a rebase, @BenjaminSchaaf

**Comment by zcorpan** [comment]
I think we can just merge this, @Ms2ger ?

**Pull request merged**